### ISSUE TITLE
Message to let users know about chain registration

### ISSUE THREAD

**Issue by marcelovani** [hechoendrupal/drupal-console-core#283]
We have a new feature that is probably unknown to many users
Chains can contain metadata that make them discovered and registered so they can be executed as normal commands.

Example: ~/.console/chain/test.yml
```
# Usage
# drupal test:foo
# drupal test:foo --project="alpha"
# drupal test:foo --project="alpha" --directory="/tmp"
command:
  name: test:foo
  description: 'Example of chain with metadata'
vars:
  project:
    - beta
    - omega
commands:
  # This chain shows the variables passed either via placeholders or interactive mode
  - command: exec
    arguments:
      bin: 'echo "This is a test: Project:%{{project}} Directory:%{{directory}}"'
```

If you run `drupal list` you will see the `test:foo` command listed

- We need to add a yellow box with a notice to tell users that they should add the metadata to their chain commands and no longer use the `--file` argument.
- The message should also mention that the `--file` argument will be deprecated on 2.0 release
- The message should no longer be displayed if the metadata is implemented on the yml files
- We need to create a sub-case to update the documentation

**Comment by jmolivas** [comment]
Fixed with PR https://github.com/hechoendrupal/drupal-console-core/pull/297

 ![](https://monosnap.com/file/6hEj3RALWLtZynCdPKiw6WT44s2NIG.png) 

Message only show at `list` command

Translation message could be improved at:
https://github.com/hechoendrupal/drupal-console-en/blob/master/translations/chain.yml#L18-L19

**Comment by jmolivas** [comment]
This improvement was included on 1.6.0
